diff --git a/configure.ac.orig b/configure.ac
index c698818..3988fa0 100644
--- a/configure.ac.orig
+++ b/configure.ac
@@ -25,7 +25,8 @@ GETTEXT_PACKAGE=powertop
 AC_SUBST([GETTEXT_PACKAGE])
 AM_SILENT_RULES([yes])
 AM_GNU_GETTEXT([external])
-AM_GNU_GETTEXT_VERSION([0.18.2])
+AM_GNU_GETTEXT_VERSION([0.22])
+AM_GNU_GETTEXT_REQUIRE_VERSION([0.22])

 m4_ifdef([AM_PROG_AR], [AM_PROG_AR])
 AX_REQUIRE_DEFINED([AX_ADD_FORTIFY_SOURCE])

